README.rst

```diff
@@ -419,10 +419,10 @@ along with libsonata.  If not, see <https://www.gnu.org/licenses/>.
 
 
 .. |license| image:: https://img.shields.io/pypi/l/libsonata
-                :target: https://github.com/BlueBrain/libsonata/blob/master/COPYING.LESSER
+                :target: https://github.com/openbraininstitute/libsonata/blob/master/COPYING.LESSER
 
-.. |coverage| image:: https://coveralls.io/repos/github/BlueBrain/libsonata/badge.svg
-                 :target: https://coveralls.io/github/BlueBrain/libsonata
+.. |coverage| image:: https://coveralls.io/repos/github/openbraininstitute/libsonata/badge.svg
```

Comment:
I guess this needs to be setup?

Reply:
no idea :D

Reply:
I do not think I have the credential of the account... How is coverall set up exactly?

Reply:
prob. the admin of this; which I appear to be, who knew.